src/components/layout/Navbar.js

```diff
@@ -67,23 +67,25 @@ class Navbar extends Component {
               </div>
             </form>
 
-            <ul className="navbar-nav ml-auto col-2">
-              <li className="nav-item">
-                <LoginModal
-                  buttonLabel="Login"
-                  title="Login"
-                  actionLabel="Login"
-                />
-              </li>
-              <li className="nav-item">
-                <Link
-                  className="btn more-rounded hover-w-b btn-sm my-2 my-sm-0 mr-sm-2"
-                  to="/signup"
-                >
-                  Sign Up
-                </Link>
-              </li>
-            </ul>
+            <div className="navbar-buttons">
```

Comment:
rather than doing
padding-top: 14px;

you can put in the class name
div className="navbar-buttons pt-14"

That's bootstraps notation for margins/padding
p (padding)
t (top)
-
14 (how high/low/etc you want it)